### ISSUE TITLE
user that has no following can't login.

### ISSUE THREAD

**Issue by erqan** [ArcoMul/Twisting#38]
it suspends at **updating followers** screen.

**Comment by ArcoMul** [comment]
Doesn't every account by default follow `@twister`? Under that assumption I didn't take in account that there are account who follow no-one, have 0 following.

**Comment by erqan** [comment]
No, twister-html does that, follows automatically twister. and user can
unfollow as well.

15-10-2015 17:43 tarihinde Arco yazdı:

**Comment by ArcoMul** [comment]
I see, didn't know that. I'll have to figure something out then

The problem is that the `getfollowing` call sometimes returns an empty array even though the user is following people. So on the _updating followers_ screen it polls that call until it returns a not-empty array.

**Comment by erqan** [comment]
yes, at starting it requires some relaxation time... maybe waiting at
background would be better.
